doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -241,33 +325,33 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
```

Comment:
There are following new known issues to be added for the Matter:
```suggestion
The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-5-0

KRKNWK-17864: When using Wi-Fi Low Power Mode, the communication with the device may not work after it re-connects to the newly respawned Wi-Fi network
  The communication with Matter over Wi-Fi device sometimes does not work after it re-connects to the Wi-Fi network.
  The issue is visible in case of re-connection to the newly respawned Wi-Fi network, triggered by rebooting the Wi-Fi access point.
  The problem was not observed for re-connection triggered by loosing the connection for other reasons than Wi-Fi access point reboot.
  The issue root cause is not known, but it is highly probable to be related with the usage of Wi-Fi Low Power Mode.
  After the application reboot, the device is always able to re-connect to Wi-Fi network and operate normally.

  **Affected platforms:** nRF7002

  **Workaround:** Disable Wi-Fi Low Power mode for your application by setting :kconfig:option:`CONFIG_NRF_WIFI_LOW_POWER` to ``n`` in the application :file:`prj.conf`.
  
.. rst-class:: v2-5-0

KRKNWK-17925: The nRF Toolbox application for iOS cannot control :ref:`matter_lock_sample` using NUS
  The nRF Toolbox application sends one additional character in all NUS commands, so they are not correctly parsed by the :ref:`matter_lock_sample`.
  The issue was observed only on the nRF Toolbox 5.0.9 version on iOS system. 

  **Workaround:** Use nRF Toolbox for iOS version other than 5.0.9 or any version of nRF Toolbox for Android.

.. rst-class:: v2-5-0

KRKNWK-17914: The links to Kconfig options from :file:`Kconfig.features` do not work in |NCS| documentation 
  The links to all Kconfig options defined in :file:`modules/lib/matter/config/nrfconnect/Kconfig.features` file do not work in documentation. 
```

Reply:
Updated. Please check.